package.json

```diff
@@ -8,7 +8,9 @@
     "test": "jest --watch",
     "test:ci": "jest",
     "heroku-postbuild": "npm run build",
-    "lint": "eslint --fix --ext .js resources/assets"
+    "lint": "eslint --fix --ext .js resources/assets",
+    "docs:prepare": "gitbook install",
+    "docs:watch": "npm run docs:prepare && gitbook serve"
```

Comment:
Oh nice! I didn't think to add something like this in the Phoenix repo. I was thinking of keeping it separate but I like this. Good idea @sbsmith86 👍

Reply:
it's very nice for local dev!